task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -59,6 +59,13 @@ spec:
     description: The image is built from this commit.
     name: COMMIT_SHA
     type: string
+  - default: repos.d
+    description: Path in the repository in which yum repodir files are stored
+    name: REPOSDIR_SRC
+  - default: /etc/yum.repos.d
```

Comment:
Generally, this is the default standard directory containing YUM repo configuration files. What cases require this customization to use an alternative directory?

Reply:
Once we mount the repos to that default location, we hide the content of that directory as stored on the base image. Suppose that the user wants to have access to that configs from the base image, they can define a different target dir, and then have the dockerfile move things around so that both the original content and the custom repos are available.